docs/Cerberus.md

```diff
@@ -0,0 +1,39 @@
+# Cerberus
+
+[Cerberus](https://www.cl.cam.ac.uk/~pes20/cerberus/) offers executable semantics for a substantial fragment of C and
+CHERI-C languages. It is implemented via an elaboration into a simpler Core language, which is displayed as the compiler
+output in the Compiler Explorer. Evaluation of C programs (execution) is also supported.
+
+## Prerequisites
+
+The easiest way to install both the Cerberus and Cerberus-CHERI compilers is by using Docker:
+
+`docker pull vzaliva/cerberus-cheri`
+
+Then, for example, you can print the _Core_ elaboration for `test.c` using ISO C semantics:
+
+`docker run -v $HOME/tmp:/mnt -it vzaliva/cerberus-cheri cerberus --pp=core --exec /mnt/test.c`
+
+## Configuration
+
+The file `etc/config/c.defaults.properties` defines a group of two compilers: 'cerberus' for ISO C and 'cerberus-cheri'
```

Comment:
You could add a section here describing what's needed to configure Cerberus compilers rather than changing the defaults.